irohad/main/tls_keypair.cpp

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "main/tls_keypair.hpp"
+
+#include <fstream>
+#include <sstream>
+
+namespace {
+  std::string readFile(const std::string &path) {
+    std::ifstream file(path);
+    std::stringstream ss;
+    ss << file.rdbuf();
+    return ss.str();
+  }
+}
+
+TLSKeypair::TLSKeypair(const std::string &path)
+    : certificate(readFile(path + ".crt")),
```

Comment:
Basically these two `readFile` calls can be done somewhere in irohad.cpp, then two `Blob` objects are initialized from `readFile` result, and `TLSKeypair` is initialized from two `Blob` objects. This will allow to initialize `ServerRunner` not only from files.

Reply:
Oh, okay, nevermind the last comment, this seems more reasonable, will do so